cmd/kubetoken/kubetoken.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubetoken
+
+import (
+	"fmt"
+
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+func KubeToken() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
Product question. Should we hide it? I mean, it is a command that it is being useful to be used by `kubectl` but not sure it provides value to be executed standalone. so not sure if it makes sense to list it or not within the list of available commands.

I don't have any preference, asking just in case

Reply:
yeah that makes sense, we don't want to promote the user into running this and hiding it will help readability when running `okteto --help`